pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "app-decenterai-com"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+packages = [{include = "app"}]
```

Comment:
we don't have an app package remove this

Reply:
@RedValis i do see it now

Reply:
kindly remove {include="app"} and leave it as []

Reply:
Sure got it

Reply:
Done